packages/ocs/dashboards/persistent-internal/capacity-trend-card/capacity-trend-card.tsx

```diff
@@ -50,6 +84,29 @@ const CapacityTrendCard: React.FC = () => {
   const { namespace: clusterNs } = useParams<ODFSystemParams>();
   const ocsCluster = systemFlags[clusterNs]?.ocsClusterName;
 
+  const [configData, configLoaded, configLoadError] =
+    useK8sWatchResource<ConfigMapKind>({
+      kind: ConfigMapModel.kind,
+      namespace: 'openshift-monitoring',
+      name: 'cluster-monitoring-config',
+      isList: false,
+    });
+
+  let retentionPeriod: number = 17;
```

Comment:
define a const under `constants` directory as: `OCP_DEFAULT_RETENTION_PERIOD = 15`
```suggestion
  let retentionPeriod: number = OCP_DEFAULT_RETENTION_PERIOD;
```